.github/workflows/push.yml

```diff
@@ -8,6 +12,17 @@ on:
       - 'v*'
 
 jobs:
+
+  old-go:
+    name: Ensure immudb compiles with the oldest supported go version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v1
```

Comment:
most probably does not make any difference, but there is already v3 https://github.com/actions/setup-go